src/modules/LanItems.rb

```diff
@@ -2336,6 +2336,8 @@ def createS390Device(rule)
     #  @param [Array] list of hashes describing interfaces in AY profile
     #  @return [Array] list of hashes for udev rules
     def createUdevFromIfaceName(interfaces)
+      return [] if !interfaces || interfaces.empty?
```

Comment:
yes, but on other hand, sanitizing input allows you to e.g. log what happen (empty interfaces) and it is easier to see that empty interfaces is same as if there is no specific device type with id or bus identifier type.